src/ducks/ui/listevisning-selectors.ts

```diff
@@ -168,5 +168,7 @@ export function getMuligeKolonner(filtervalg: FiltervalgModell, oversiktType: Ov
         .concat(addHvis(Kolonne.TOLKEBEHOV, tolkBehovErValgt()))
         .concat(addHvis(Kolonne.TOLKEBEHOV_SPRAAK, tolkBehovErValgt()))
         .concat(addHvis(Kolonne.TOLKEBEHOV_SIST_OPPDATERT, tolkBehovErValgt()))
+        .concat(addHvis(Kolonne.CV_KAN_DELES_STATUS, filtervalg.stillingFraNavFilter.length !== 0))
+        .concat(addHvis(Kolonne.CV_SVARFRIST, filtervalg.stillingFraNavFilter.length !== 0))
```

Comment:
Her burde jeg kanskje ha en nullsjekk på stillingFraNavFilter? For den vil kanskje feile hvis stillingFraNavFilter er null? ...

Reply:
stillingFraNavFilter er forøvrig en liste som inneholder to stringer dersom vi har fått inn data på det. Det første feltet vil være "CV_KAN_DELES_STATUS_JA", det andre vil være en dato (men som en string)

Reply:
1. I utgangspunktet skal det ikkje vere nødvendig med nullish-sjekk her siden vi har definert `stillingFraNavFilter` som non-nullable (non-undefinedable) i `FiltervalgModell` i `model-interfaces.ts`. 
2. Dessuten ser det ut til at propertien alltid er initialisert, ref: `filtrering.ts` linje 74. 

#2 er ikkje en god nok garanti i seg sjølv, men siden vi har #1 bør det vere greit. Dersom vi ikkje hadde #1 så måtte vi gjort ein form for sjekk.

Reply:
Etter å ha tenkt meg litt om så bør ein sjekke korleis dette filteret er typet opp i backenden. Dvs. om backenden bruker Java og det ikkje settes en default verdi (feks. tom liste) for tomme filter så må vi vel egentlig forvente at alle filter potensielt kan vere undefined og sjekke dette eksplisitt som du er inne på.

Eg tok derimot en rask kikk (i `veilarbfilter`) og ser at vi setter default til å vere tom liste der:

```java
@JsonSetter(nulls = Nulls.AS_EMPTY)
private List<String> stillingFraNavFilter = emptyList();
```